sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/ChangeFeedProcessorTest.java

```diff
@@ -508,7 +508,7 @@ public void staledLeaseAcquiring() throws InterruptedException {
 
     @Test(groups = { "simple" }, timeOut = 50 * CHANGE_FEED_PROCESSOR_TIMEOUT)
     public void readFeedDocumentsAfterSplit() throws InterruptedException {
-        CosmosAsyncContainer createdFeedCollectionForSplit = createLeaseCollection(FEED_COLLECTION_THROUGHPUT_FOR_SPLIT);
+        CosmosAsyncContainer createdFeedCollectionForSplit = createFeedCollection(FEED_COLLECTION_THROUGHPUT_FOR_SPLIT);
```

Comment:
Good catch! Also can you please add the following check at line 604 (after stopping the changeFeedProcessor:
`
            int leaseCount = changeFeedProcessor.getCurrentState()
                .map(List::size).block();
            assertThat(leaseCount > 1).as("Found %d leases", leaseCount).isTrue();
`

Reply:
Fixed